.github/workflows/build_and_test.yml

```diff
@@ -67,12 +79,25 @@ jobs:
         uses: lukka/get-cmake@latest
         with:
           cmakeVersion: 3.22.6
+
       - name: Use cmake
         run: cmake --version
 
+      - name: Install ccache
+        shell: pwsh
+        run: |
+          pwsh -File "src\fastdds_python\.github\ci\ccache.ps1"
+          $ccache_dir = $pwd.PATH
+          $ccache_dir = "$ccache_dir\.github\ccache"
+          echo "$ccache_dir" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+          Set-Item -Force -Path "env:PATH" -Value "$ccache_dir;$env:PATH"
+          ccache --version
+
       - name: Setup ccache
         uses: Chocobo1/setup-ccache-action@v1
```

Comment:
It would be nice to include this action into eProsima-CI external actions.

Reply:
Done in 2899e3427dbebfc5021439914e7b2e6b16aea818